### ISSUE TITLE
Fullscreen Windows move cursor

### ISSUE THREAD

**Issue by Hyderman** [wez/wezterm#3642]
### What Operating System(s) are you seeing this problem on?

Windows

### WezTerm version

20230408-112425-69ae8472

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

No, and I'll explain why below

### Describe the bug

First problem, the fullscreen button of Windows doesn't work.

Second problem, when using Powershell (7.3.4) and using the shortcut windows key + upArrow, it moves the cursor. It happens if there is other command before but it depends on the command sometime it don't bug.
![image](https://user-images.githubusercontent.com/81628944/235311215-b5c08e97-4416-4b51-8b3c-be85cc677ec3.png)
Fullscreen:
![image](https://user-images.githubusercontent.com/81628944/235311252-026b29b8-ba48-4e1e-9b8b-a01a93313d4b.png)

Other example, I have type 3 types a random command to have the bug:
![image](https://user-images.githubusercontent.com/81628944/235311478-e9308358-bc8a-4189-bf8c-e299e290b849.png)
Fullscreen:
![image](https://user-images.githubusercontent.com/81628944/235311499-302910ff-47f2-4420-a7eb-2909c1660e11.png)

### To Reproduce

Write a command for example "gcm gal" then press windows + upArrow.

### Configuration

Empty config don't change anything

**Comment by Hyderman** [comment]
The fullscreen button problem can be solve with "config.window_decorations = "INTEGRATED_BUTTONS|RESIZE"".

**Comment by wez** [comment]
I think you mean the maximize button rather than fullscreen? If so, that is resolved in:
* https://github.com/wez/wezterm/issues/3499

**Comment by Hyderman** [comment]
Yes thx. Also I tried with PowerShell core and windows PowerShell, both have the bug. I have no problem with CMD and wsl.
